<launch>
<node pkg="perception_neuron_tf_broadcaster" name="arm_velocity2trajectory_controllers_switch" type="arm_velocity2trajectory_controllers_switch" output="screen" >
<param name="left_arm" value="false" type="bool" />
<param name="right_arm" value="true" type="bool"/>
</node>
</launch>
